lib/fittings/swagger_render.js

```diff
@@ -6,6 +6,7 @@ var injector = require('../../index.js').injector;
 var _ = injector.get('_');
 
 module.exports = function create(fittingDef) {
+    injector = require('../../index.js').injector;
```

Comment:
@yyscamper This change is a work-around for an issue with the way mocking is being done.  Without this line, the http server will run with an old version of the injector that still has mocks from the previous test.  We have work planned to make structural changes to all of the unit tests to avoid the need for this work-around in the future.